com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -824,5 +824,13 @@ protected NetworkObject GetNetworkObject(ulong networkId)
         {
             return NetworkManager.SpawnManager.SpawnedObjects.TryGetValue(networkId, out NetworkObject networkObject) ? networkObject : null;
         }
+
+//        public void OnDestroy()
```

Comment:
This is the last issue I need to resolve; I *really* hoped this would work as it would make cleanup automatic.